lib/players/human.rb

```diff
@@ -0,0 +1,12 @@
+module Players
+
+  class Human < Player
+
+    def move(board)
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  board.display
  print "Enter move (1-9):"
  gets.chomp
end
```